coach/src/user/model.coffee

```diff
@@ -50,7 +57,13 @@ User =
     Boolean(@isLoggedIn() and @getCourse(collectionUUID))
 
   getCourse: (collectionUUID) ->
-    _.findWhere( @courses, ecosystem_book_uuid: collectionUUID )
+    _.chain(@courses)
+      .where( ecosystem_book_uuid: collectionUUID )
+      .sortBy( (course) ->
+        course.getRole().joined_at
+      )
+      .last()
+      .value()
 
```

Comment:
This is the bug fix - before we were just looking at the first match, now we look at the most recently joined